btree/btree.c

```diff
@@ -1433,19 +1433,25 @@ enum lru_used_space_watermark{
  * Watermarks for BE space occupied by nodes in lru list.
  */
 /** LRU purging should not happen below low used space watermark. */
-int64_t lru_space_wm_low    = LUSW_LOW;
+static int64_t lru_space_wm_low;
 
 /**
  * An ongoing LRU purging can be stopped after reaching target used space
  * watermark.
  */
-int64_t lru_space_wm_target = LUSW_TARGET;
+static int64_t lru_space_wm_target;
 
 /**
  * LRU purging should be triggered if used space is above high used space
  * watermark.
  */
-int64_t lru_space_wm_high   = LUSW_HIGH;
+static int64_t lru_space_wm_high;
+
+/**
+ * LRU purging should be triggered if used space is above high used space
+ * watermark.
+ */
+static int64_t set_lrulist_trickle_release;
```

Comment:
s/static int64_t set_lrulist_trickle_release/static bool lrulist_trickle_release

Reply:
I will match the above parameter to what was set in your PR.

Reply:
Done.